apache-rat-core/src/main/java/org/apache/rat/DeprecationReporter.java

```diff
@@ -83,18 +83,6 @@ public static void logDeprecated(final Class<?> clazz) {
         }
     }
 
-    /**
-     * Log deprecated options
-     * @param log The log to write to.
-     * @param option potentially deprecated option to log.
-     */
-    // TODO remove this when commons-cli 1.7.1 or higher is available
-    public static void logDeprecated(final Log log, final Option option) {
-        if (option.isDeprecated()) {
-            getLogReporter(log).accept(option);
-        }
-    }
-
```

Comment:
Could you clarify how the new commons-cli is related to RAT's deprecated options? or is there no connection .....

Reply:
Thanks for the clarification. I added a changelog entry on the branch.